src/main/webapp/js/user-page-loader.js

```diff
@@ -42,6 +42,9 @@ function showMessageFormIfViewingSelf() {
             loginStatus.username == parameterUsername) {
```

Comment:
Oh, I see. It checks for the user here. However, I think it can be exploited by simply removing the class from the #about-me-form

Reply:
By the way, storing user's about-me is not dependent on request's 'user' parameter.

`String userEmail = userService.getCurrentUser().getEmail();`

When X sends post request from Y's about-me, it changes about-me of X.

Reply:
I see. I thought it was getting the user from the request